parla/task_runtime.py

```diff
@@ -57,7 +57,10 @@ def clear_dependencies(self):
         self.dependencies = None
 
     def __repr__(self):
-        return "TaskRunning({}, {}, {})".format(self.func.__name__, self.args, self.dependencies)
+        if self.func:
+            return "TaskRunning({}, {}, {})".format(self.func.__name__, self.args, self.dependencies)
+        else:
+            return "Functionless task"
```

Comment:
This is good lol

Reply:
I was using the logger for debugging and it kept complaining to me about trying to get `None.__name__` haha